pyrs/core/summary_generator_stress.py

```diff
@@ -158,13 +156,15 @@ def _write_field_3d(row: int, field: str):
                         strain_field = strain.field
```

Comment:
I would rather use the facade object, because it provides `ScalarFieldSample` objects
```
facade.selection = direction
strain_field = facade.strain
```